README.md

```diff
@@ -1,12 +1,27 @@
-# pie_of_the_month
-Google Chrome Extension to show what the Pinthouse pizza of the month currently is. For more information about Pinthouse pizza, please visit http://pinthousepizza.com/burnet/about.
+# Pie Of The Month
 
-File templates and info used to write this .md file from https://gist.github.com/PurpleBooth/109311bb0361f32d87a2#file-readme-template-md and https://github.com/gorhill/uBlock.
+> Google Chrome Extension to show the current Pinthouse pizza of the month.
 
 ## Prerequisites
+
 This extension was tested during development in Google Chrome. Run this in Chrome for best results.
 
+## Installation
+
+1. Clone this repo (extract if needed)
+2. Go to chrome extensions [chrome://extensions](chrome://extensions)
+3. Enable developer mode
+4. Click on load unpacked extension and select the extension folder.
+
+For more information see the Chrome Developer getting started [Tutorial](https://developer.chrome.com/extensions/getstarted)
+
+## Related
+Information about [Pinthouse pizza](http://pinthousepizza.com/burnet/about)
+
 Research suggests that it may be possible to install and run this extension in Firefox per https://www.howtogeek.com/233355/how-to-install-extensions-from-outside-the-chrome-web-store-and-firefox-add-ons-gallery/. I have not yet tested this case and in case I end up doing so, I will update this project with the results.
 
-## Installation
-Download and unzip the zipped file, then, follow Google instructions for installing the extension at https://developer.chrome.com/extensions/getstarted.
+## Sources:
+File templates and info used to write this readme.md  https://gist.github.com/PurpleBooth/109311bb0361f32d87a2#file-readme-template-md and https://github.com/gorhill/uBlock.
+
+## Legal
+❗️ This extension is not sponsored or endorsed by Pinthouse Pizza. 
```

Comment:
Again, a very nice change and thank you for taking initiative!

NOTE TO SELF: At some point, I may want to go back and make some more changes/additions to these sections for purposes of customization.

Reply:
I need to add the source for much of the inspiration for these changes. 
https://github.com/softvar/enhanced-github/blob/master/README.md

Reply:
Just fixed the issues in the latest commits.